.github/workflows/update_documentation.yml

```diff
@@ -7,15 +7,13 @@ on:
 jobs:
   create_pull_request:
     runs-on: ubuntu-latest
+    container: upciti/wakemebot:1.3.0
```

Comment:
This image is quite large, this would be slower and consume more bandwidth

We can add wakemebot to wakemeops and install it with apt but which component should we add it to?

Reply:
I was thinking more about time than bandwidth.

Maybe `Terminal`? or with `ops2deb`?

but are you sure about `1.3.0`, not`main`?